testing/infrastructure/ComponentShape.ttl

```diff
@@ -1,108 +0,0 @@
-@prefix owl: <http://www.w3.org/2002/07/owl#> .
```

Comment:
Could you please explain why this file was deleted? Thanks!

Reply:
This is a duplicate with the InfrastructureComponentShape, I think.